build.sbt

```diff
@@ -1,14 +1,19 @@
 val catsVersion = "2.7.0"
 
-ThisBuild / tlBaseVersion := "21.8" // TODO
+ThisBuild / tlBaseVersion := "21.8"
+ThisBuild / tlMimaPreviousVersions := Set.empty
```

Comment:
Once we publish this under `org.typelevel`, we can set this value and start checking bincompat, at least within this library.

Reply:
With some fiddling, hopefully it can set itself automatically via tags.

Reply:
That would be nice! Do you mean fiddling here or in sbt-typelevel?

Reply:
Probably here. I purposely wanted to support only early-semver in sbt-typelevel to keep things simple(r).

I think the real question is how far back do you want to keep bincompat? Just per month-year, or something more ad-hoc?